edx_solutions_api_integration/users/views.py

```diff
@@ -753,6 +753,18 @@ def post(self, request, user_id):  # pylint: disable=R0915
             existing_user.username = username
             response_data['username'] = existing_user.username
             existing_user.save()
+            if old_username != username:
+                storage = get_profile_image_storage()
+                profile_image_names = get_profile_image_names(old_username)
+                new_profile_image_names = get_profile_image_names(username)
+                for old_image_size, old_image_name in profile_image_names.items():
+                    if storage.exists(old_image_name):
+                        old_image_path = os.path.join(storage.location, old_image_name)
+                        new_image_path = os.path.join(storage.location, new_profile_image_names[old_image_size])
+                        try:
+                            os.rename(old_image_path, new_image_path)
```

Comment:
S3 boto storage does not support renaming files. We need to copy the file to new path and removing the old one.

ref: https://stackoverflow.com/questions/2481685/amazon-s3-boto-how-do-you-rename-a-file-in-a-bucket

Reply:
Done